front/public/locales/fr/stdcm.json

```diff
@@ -26,7 +26,7 @@
     },
     "stopCalculation": "Arrêter le calcul"
   },
-  "spaceSpeedGraphic": "Graphique Espace-Vitesse",
+  "speedSpaceChart": "Graphique Espace Vitesse",
```

Comment:
Maybe you can change it also for the `.en` file for harmonization.

Reply:
yes sure

Reply:
my bad, I missed it, it's done